src/hclib.c

```diff
@@ -329,9 +343,9 @@ void forasync1D_flat(void * forasync_arg) {
         printf("Scheduling Task %d %d\n",low0,high0);
```

Comment:
Huh? I didn't add that print statement. In any case, it's protected by DEBUG_FORASYNC so it is normally disabled.